Samples/kotlin/src/main/res/layout/dialog_offroute.xml

```diff
@@ -12,11 +13,11 @@
         android:layout_height="wrap_content"
         android:fontFamily="sans-serif-medium"
         android:textColor="@color/azure"
-        android:textSize="26.67dp"
+        android:textSize="22sp"
         android:paddingLeft="25dp"
```

Comment:
I realize this isn't part of your changeset, but Can we change paddingLeft/paddingRight to paddingStart/PaddingEnd for every occurence in the layouts modified?

Reply:
Can do, it will give an additional lint warning though after the change, saying that I dont have a paddingStart. Should I set to paddingStart to 0dp? or ignore the warning?

Reply:
In those situations we should add symmetrical padding. (i.e. if we are just specifying end, we should add a matching start)

Reply:
got it, added symmetrical padding

Reply:
definitely got it this time